javascript/javascript2/week2/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Document</title>
+</head>
+<body>
+ 
+
+<script src="doublingOfNumbers.js"></script>
+<script src="Working with movies.js"></script>
```

Comment:
In general, it's best to name files on the web without spaces. 
So workingWithMovies.js, or working_with_movies.js, or working-with-movies.js as examples